src/main/java/guru/qa/allure/notifications/template/MessageTemplate.java

```diff
@@ -22,25 +23,21 @@ public class MessageTemplate {
 
     private final TemplateConfig templateConfig = new TemplateConfig();
 
-    public String of(String templateFile) {
+    public String of(String templateFile) throws MessageBuildException {
         LOG.info("Processing template {}", templateFile);
         Template template = null;
         try {
             LOG.info("Parsing template");
             template = templateConfig.configure().getTemplate(templateFile);
         } catch (IOException ex) {
-            LOG.info("Unable to parse template {}! Reason: {}", templateFile,
-                    ex.getMessage());
-            System.exit(1);
+            throw new MessageBuildException(String.format("Unable to parse template %s!", templateFile), ex);
```

Comment:
Please refactor to avoid copy-pasted logic

Reply:
it will be done in another PR